aten/src/ATen/native/sparse/cuda/cuSPARSELtOps.cpp

```diff
@@ -103,14 +91,16 @@ at::Tensor _cslt_compress(const Tensor& sparse_input)
     return compressed_tensor;
 }
 
-std::tuple<int64_t, at::Tensor> _cslt_sparse_mm_impl(
+std::tuple<at::Tensor, int64_t, int64_t, bool, int64_t> _cslt_sparse_mm_impl(
```

Comment:
not sure where to put this.

1. A/B=int8 C/D=int8 and A/B=E4M3 C=BF16 D=E4M3 are missing.
2. cuSPARSELt supports ReLU (for all data types) and GeLU (only a few data types). It seems that they are not suported in `_cslt_sparse_mm_impl()`.

Reply:
For (1) I don't think this matters too much ATM since we don't support passing C, we just use the zero matrix we intialize for D. Is this something that you find makes a big difference for perf/acc reasons? 

For (2) I think it makes sense to add in a later PR, as we should add a inductor lowering to fuse the relu when possible when torch.compiling.